frontend/catalyst/pennylane_extensions.py

```diff
@@ -1929,10 +1934,21 @@ def circuit(x: float):
     wires = list(wires) if isinstance(wires, (list, tuple)) else [wires]
```

Comment:
Could you add an example with postselect in the example section of the documents?

Reply:
Done